extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/ControllerImpl.java

```diff
@@ -929,6 +941,7 @@ public Map<String, TaskReport> liveReports()
     return TaskReport.buildTaskReports(
         new MSQTaskReport(
             id(),
+            null,
```

Comment:
Can you add some commentary here around why tags should not be included in the live reports?